clarity/contracts/token/token-wstx.clar

```diff
@@ -85,15 +86,16 @@
 ;; This can only be called by recipient since stx-transfer is involved ;; tx-sender -> .alex-vault
 (define-public (mint (recipient principal) (amount uint)) 
   (begin
-    (try! (stx-transfer? amount recipient .alex-vault))
+    (asserts! (is-eq tx-sender recipient) ERR-NOT-TOKEN-OWNER)
```

Comment:
Would you please update GitBook (https://app.gitbook.com/invite/-MbzMWHeiZPWiQbg7qqk/xPWMh6jZJ3xhVcVCXlza) when you add new ERR codes?